com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStream.cs

```diff
@@ -9,15 +9,69 @@
 
 namespace UnityEngine.Formats.Alembic.Importer
 {
-    internal sealed class AlembicStream : IDisposable
+    sealed class AlembicStream : IDisposable
     {
-        public struct UpdateSamplesJob : IJob
+        /// <summary>
+        /// This class hides the context. The reason for that it that there are Jobs depending on it and we need
+        /// to ensure they get completed before any set happens on the context
+        /// </summary>
+        public struct ContextPair
```

Comment:
Can I suggest SecureContext or SyncedContext? I think it conveys more meaning than ContextPair

Reply:
changed to SafeContext.